polytracker/src/dfsan_rt/dfsan/dfsan_log_mgmt.cpp

```diff
@@ -280,13 +281,19 @@ void taintManager::addJsonRuntimeTrace() {
         }
         const auto& taints = trace.taints(bb);
         if (!taints.empty()) {
-          // dfsan_labels are 1-indexed
-          std::vector<dfsan_label> zeroIndexed;
-          zeroIndexed.reserve(taints.size());
+          std::vector<int> byteOffsets;
```

Comment:
Question: Does the `taints` set only have taint labels that are canonical?

Reply:
Correct